apps/src/javalab/JavalabEditorTabMenu.jsx

```diff
@@ -27,19 +31,43 @@ export class JavalabEditorTabMenu extends Component {
     const {
       renameFromTabMenu,
       deleteFromTabMenu,
+      moveTabRight,
+      moveTabLeft,
       showVisibilityOption,
       changeFileTypeFromTabMenu,
       fileIsVisible,
-      fileIsValidation
+      fileIsValidation,
+      activeTabKey,
+      orderedTabKeys
     } = this.props;
+
     let elements = [
       <button onClick={renameFromTabMenu} key="rename" type="button">
         {javalabMsg.rename()}
-      </button>,
+      </button>
+    ];
+    const tabsLength = orderedTabKeys.length;
+    const index = orderedTabKeys.indexOf(activeTabKey);
+    if (index > 0) {
+      elements.push(
+        <button onClick={moveTabLeft} key="moveLeft" type="button">
+          {javalabMsg.moveLeft()}
+        </button>
+      );
+    }
+    if (index < tabsLength - 1) {
+      elements.push(
+        <button onClick={moveTabRight} key="moveRight" type="button">
+          {javalabMsg.moveRight()}
+        </button>
+      );
+    }
```

Comment:
If there is only one file visible to the student but there are additional support/validation files, are moveLeft/Right still hidden?

Reply:
Yes. Screenshot below of sample Java Lab level with validation files and only one visible file. Thanks for checking!
<img width="940" alt="Screen Shot 2022-09-16 at 2 32 28 PM" src="https://user-images.githubusercontent.com/107423305/191598527-e25201c7-56a1-402a-a43f-8eee0fa88631.png">